tests/test_seq2pat.py

```diff
@@ -1301,7 +1302,7 @@ def test_repeated_calls_seq2pat(self):
         sequences = [[11, 12, 13], [11, 12, 13, 14]]
 
         # Sequential pattern finder
-        seq2pat = Seq2Pat(sequences)
+        seq2pat = Seq2Pat(sequences, batch_size=2)
```

Comment:
What is the default batch_size? Is behavior same as before?

Reply:
`batch_size=None` by default, so it is running on entire set on a single thread as before. When it is set, the mining will be enabled to run on `batch_size` of sequences in parallel.

Reply:
n_jobs default is set to 2, no? Is this still same as before?

Reply:
Btw, not sure why this test is changing?

Reply:
This test has been covered in test_seq2pat_batch and the original test should be the same as before.  This file should keep unchanged.